languages/javalang/builder-renderer/commercetools-java-client/src/main/java/client/HttpResponse.java

```diff
@@ -0,0 +1,38 @@
+package client;
+
+public class HttpResponse {
+    
+    private int statusCode;
+    private HttpHeaders headers;
+    private String body;
+
+    public HttpResponse(int statusCode, HttpHeaders headers, String body) {
```

Comment:
body might also be binary, i think that a byte[], or a stream would be more appropriate.

Reply:
My idea is to keep it as a string to hold jsons. When we start implementing image upload and so on we can see how it can be improved to support other types. Keep it as simple as possible until we have to make it more complex. What do you think about that ?

Reply:
sounds good to me 👍